04/5_formatting/print_data_units.c

```diff
@@ -0,0 +1,94 @@
+#include <stdio.h>
+
+#include "print_data_units.h"
+
+#define S_SCALE (1024.0f)
+
+#define S_BYTE (1.0f)
+#define S_KB (S_SCALE)
+#define S_MB (S_SCALE * S_KB)
+#define S_GB (S_SCALE * S_MB)
+#define S_TB (S_SCALE * S_GB)
+#define S_PB (S_SCALE * S_TB)
+
+void print_byte_conversion_chart(void)
+{
+    double divisor = 1.0f;
+
+    printf("%26s %17s %17s %17s %17s %17s\n", 

```

Comment:
now thinking about it.. can we not print all these in for loop?

the only thing that changes is row names.. which can be in a char* array